dashboard-app/src/util/mathUtil.ts

```diff
@@ -0,0 +1,3 @@
+export const roundToDecimal = (n: number) => (Math.round(n * 100) / 100);
```

Comment:
"decimal" is being used to mean "2dp" here. Can we not just make this generic?
```ts
const roundTo = (n: number, p: number) => Math.round(n * (10 ** p)) / (10 ** p);
```
Then it can be moved to `twine-util`. As it is it encodes a specific assumption about how numbers should be presented in the data tables.